Sources/PhantomConnect/Views/PhantomConnectViewModel.swift

```diff
@@ -22,7 +22,13 @@ public class PhantomConnectViewModel: ObservableObject {
     // MARK: - Public API
     
     // MARK: Public Properties
-   
+
+    /// Saving keys in appstorage(user defaults).
+    /// THIS IS NOT SECURE! FOR DEMO ONLY!
+    /// Keychain can be used as a ssecure storage
+    @AppStorage("k_dap_secret_key") var secretKey: Data?
```

Comment:
I would rather pass these values into the init method rather than have them be part of the core code. Then this implementation could simple be moved over to the example app. We could just have a convenience init method that is purely for testing purposes. Thought?